pkg/wpa/passphrase/passphrase_test.go

```diff
@@ -0,0 +1,103 @@
+package passphrase
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+)
+
+type RunTestCase struct {
+	name  string
+	essid string
+	pass  string
+	out   []byte
+	err   error
+}
+
+var (
+	essidStub     = "stub"
+	shortPass     = "aaaaaaa"                                                          // 7 chars
+	longPass      = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" // 64 chars
+	validPass     = "aaaaaaaaaaaaaaaa"                                                 // 16 chars
+	correctOutput = []byte(
+		`network={
+	ssid="stub"
+	#psk="aaaaaaaaaaaaaaaa"
+	psk=e270ba95a72c6d922e902f65dfa23315f7ba43b69debc75167254acd778f2fe9
+}
+`) // indentation matters
```

Comment:
What is this comment trying to prevent in the future?

Reply:
you need to make that comment much clearer, it already confused two people :-)

or just remove it.

Reply:
Fixed. I just removed it